mingw-w64-ansifilter/PKGBUILD

```
@@ -0,0 +1,29 @@
+# Maintainer: Wu, Zhenyu <wuzhenyu@ustc.edu>
+
+_realname=ansifilter
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=2.20
+pkgrel=1
+pkgdesc="remove or convert ANSI codes to another format (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
+url="https://gitlab.com/saalen/ansifilter"
+license=('GPL3')
```

```suggestion
license=('spdx:GPL-3.0-or-later')
```

Please use SPDX identifier for licenses https://spdx.org/licenses/